src/ImageService.Data/ImageMessageRepository.cs

```diff
@@ -1,77 +1,64 @@
-﻿using LT.DigitalOffice.ImageService.Data.Interfaces;
-using LT.DigitalOffice.ImageService.Data.Provider;
-using LT.DigitalOffice.ImageService.Models.Db;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using LT.DigitalOffice.ImageService.Data.Interfaces;
+using LT.DigitalOffice.ImageService.Data.Provider;
+using LT.DigitalOffice.ImageService.Models.Db;
+using Microsoft.Data.SqlClient;
 
 namespace LT.DigitalOffice.ImageService.Data
 {
-    public class ImageMessageRepository : IImageMessageRepository
+  public class ImageMessageRepository : IImageMessageRepository
+  {
+    private readonly IDataProvider _provider;
+    public ImageMessageRepository(IDataProvider provider)
     {
-        private readonly IDataProvider _provider;
-        public ImageMessageRepository(IDataProvider provider)
-        {
-            _provider = provider;
-        }
-
-        public List<Guid> Create(List<DbImageMessage> imagesMessages)
-        {
-            if (imagesMessages.Contains(null))
-            {
-                return null;
-            }
-
-            _provider.ImagesMessages.AddRange(imagesMessages);
-            _provider.Save();
+      _provider = provider;
+    }
 
-            return imagesMessages.Select(x => x.Id).ToList();
-        }
+    public List<Guid> Create(List<DbImageMessage> imagesMessages)
+    {
+      if (imagesMessages.Contains(null))
+      {
+        return null;
+      }
 
-        public List<DbImageMessage> Get(List<Guid> imageIds)
-        {
-            return _provider.ImagesMessages.Where(x => imageIds.Contains(x.Id)).ToList();
-        }
+      _provider.ImagesMessages.AddRange(imagesMessages);
+      _provider.Save();
 
-        public DbImageMessage Get(Guid imageId)
-        {
-            return _provider.ImagesMessages.FirstOrDefault(x => x.Id == imageId);
-        }
+      return imagesMessages.Select(x => x.Id).ToList();
```

Comment:
why we need to return guids of imagesMessages if we know them before fuction is called?

Reply:
Because we must return id's of created entities and before the function call they aren't created yet

Reply:
if its guid then it created via guid.newGuid() and we know created id before fucntion was called. Or am i wrong?

Reply:
we know its id, yes, but it's not in database yet
we return ids after saving images in db

Reply:
after saving images in db ids doesnt changed but ok.